.travis.yml

```diff
@@ -14,7 +14,8 @@ cache:
 
 env:
   global:
-    - DRIVER_VERSION=1.6.0
+    - DRIVER_VERSION=1.7.0
+    - DRIVER_BRANCH="master"
```

Comment:
Given that [`install-extension.sh`](https://github.com/mongodb/mongo-php-library/blob/master/.travis/install-extension.sh) checks for `DRIVER_BRANCH` with `if` and then `DRIVER_VERSION` via an `elif`, wouldn't defining both result in `DRIVER_VERSION` never getting used?

Reply:
Yes, which is intended until we've got a stable driver release, since PECL won't be able to install the 1.7 driver.

Reply:
A "todo" comment similar to what you added below for the new matrix configs would work well here.